assets/files.wxs

```diff
@@ -3065,6 +3065,15 @@
       <Component Id="cmpEC42F4F98D70437BB0E290A0E0FE1775" Guid="{5483092a-12c2-4e6c-b730-be491b8909c0}">
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-linux`
Breakpoint SDK Unit Tests.Handling errors while managing breakpoints in a remote runspace via the SDK.Error occurred in Context block
```powershell
Exception calling "RemoveBreakpoint" with "1" argument(s): "The background process reported an error with the following message: Remote transport error: ."
At /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1:154 char:13
```